server/controllers/userController.js

```diff
@@ -0,0 +1,82 @@
+const { User } = require('../models/index');
+
+const getAllUsers = async (req, res) => {
+    try {
+        const users = await User.findAll();
+        if (!users) {
+            res.status(404).send("Error fetching users: Users not found");
+        } else {
+            res.status(200).json(users);
+        }
+    } catch (error) {
+        res.status(500).send(error);
+    }
+}
+
+const getUser = async (req, res) => {
+    try {
+        const userId = req.params.userId
+        const user = await User.findByPk(userId);
+        if (!user) {
+            res.status(404).send(`Error fetching user: User with ID ${userId} not found`);
+        } else {
+            res.status(200).json(user);
+        }
+    } catch (error) {
+        res.status(500).send(error);
+    }
+}
+
+const createUser = async (req, res) => {
+    try {
+        const userInfo = req.body;
+        await User.create(userInfo);
+        const newUser = await User.findOne({ where: { email: userInfo.email } });
+        if (!newUser) {
+            res.status(500).json({ error: 'Error creating user '});
+        } else {
+            res.status(200).json(newUser);
```

Comment:
please change code to 201 (more correct to signals resource creation with 201 vs 200) 
complete status code list for future reference: 
https://umbraco.com/knowledge-base/http-status-codes/#:~:text=The%20201%20Created%20status%20code,multiple%20new%20resources%20being%20created.